frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -763,6 +763,76 @@ const TAGGED_TEMPLATES = {
       </>
     );
   },
+  createdDeclarationProfile: (activity: IActivity, isPremiumTier: boolean) => {
+    let teamText: React.ReactNode = "";
+    if (!isPremiumTier) {
+      teamText = "all macOS hosts";
+    } else if (activity.details?.team_name) {
+      teamText = (
+        <>
+          macOS hosts assigned to the <b>{activity.details.team_name}</b> team
+        </>
+      );
+    } else {
+      teamText = "macOS hosts with no team";
+    }
```

Comment:
Seems like this could be extracted as a helper, but feel free to leave that as a future optimization :)

Reply:
hey i replaced this with `getProfileMessageSuffix` which we have already.